### ISSUE TITLE
Installation of different/same versions of QGIS Desktop and QGIS Server on Windows 10

### ISSUE THREAD

**Issue by TerraTraveler** [qgis/QGIS#50896]
### What is the bug or the crash?

Both, I think.

I have been trying to install QGIS Desktop 3.22 LTR and QGIS Server 3.22 LTR as I was told that both need to be the same version for QGIS Server + QWC2 to function properly serve maps over the web. 

Earlier I had installed QGIS Desktop as well as QGIS Server using "advanced install" in OSGeo4W Network package, but the system always installed QGIS Desktop 3.28 Firenza and NOT  QGIS Desktop 3.22 Białowieża

So I installed QGIS Desktop 3.22 Białowieża using standalone Windows install, and it did, however, QGIS Desktop 3.22 Białowieża did not show up on the start menu.

I started it by searching for it in Cortana search (Windows search box near Start icon) and it did start QGIS Desktop 3.22 Białowieża.

I decided to open a *.qgs project file saved under 3.28 Firenza as it was also previously installed.

I received an error message that it was not able to load the entire project. And when began to work with that project, I got this panel in which I am writing the report

My system is Windows 10 Home with 16GB memory and C: drive with 400 GB of space and other HDD with several terabytes of external USB drives as well.

I *DO need to have* both QGIS Desktop, QGIS Server and QWC2 with all its services functional functional at the same time on Windows 10 Home localhost to prove all the web mapping claims made by QGIS Community!

Please help.
BJ Raval / Arizona, USA
ravalbj1@gmail.com

### Steps to reproduce the issue

1. Installed QGIS Server and associated programs and QGIS Desktop with Advanced install using OSGeo4W Network install
2. This installed QGIS Desktop 3.28 Firenza but not QGIS Desktop 3.22 LTR (There is no way to check which version of QGIS Server I was using unless I do not know where to check
3. Installed QGIS Desktop 3.22 LTR using standalone installer
4. QGIS Desktop 3.22 icon did not show up in Windows Start menu
5. Started QGIS Desktop 3.22 searching and using Cortana (Windows search box)
6. Opened a QGIS project created with QGIS Desktop 3.28 Firenza
7. QGIS crashed

### Versions

<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
<html><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><style type="text/css">
p, li { white-space: pre-wrap; }
</style></head><body>

QGIS version | 3.22.12-Białowieża | QGIS code revision | b8534cb1
-- | -- | -- | --
Qt version | 5.15.3
Python version | 3.9.5
Compiled against GDAL/OGR | 3.5.2 | Running against GDAL/OGR | 3.5.3
PROJ version | 9.1.0
EPSG Registry database version | v10.074 (2022-08-01)
GEOS version | 3.10.3-CAPI-1.16.1
SQLite version | 3.39.4
PDAL version | 2.4.3
PostgreSQL client version | 14.3
SpatiaLite version | 5.0.1
QWT version | 6.1.6
QScintilla2 version | 2.13.1
OS version | Windows 10 Version 2009
  |   |   |  
Active Python plugins
attributeBasedClustering | 2.2
awap | 2.0.1
coordinate_capture | 0.2
d3datavis | 3.0.3
dataexplorer | 3.1.1
DataPlotly | 3.9.2
GeoCoding | 2.18
gml_application_schema_toolbox | 1.4.0-beta6
Hqgis | 1.1.1
latlontools | 3.6.6
layout_loader | 1.1
lizmap | 3.9.2
mmqgis | 2021.9.10
NavTable | 0.1.1
networks | 2.7.3
nominatim_locator_filter | 0.2.4
OnlineRoutingMapper | 0.9
openlayers_plugin | 2.0.0
OSMDownloader | 1.0.3
OSMtools | 3.2
pgRoutingLayer | 3.0.2
QCopycanvas | 0.5
qdraw | 3.0.2
qgeric | 1.91
qgiscloud | 3.6.0
QuickOSM | 2.1.1
SemiAutomaticClassificationPlugin | 7.10.8
shapetools | 3.4.9
timemanager | 3.6
vector_tiles_reader | 3.2.2
VoGisProfilTool | 3.0.2
db_manager | 0.1.20
grassprovider | 2.12.99
MetaSearch | 0.3.5
processing | 2.12.99
sagaprovider | 2.12.99

</body></html>QGIS version
3.22.12-Białowieża
QGIS code revision
[b8534cb1](https://github.com/qgis/QGIS/commit/b8534cb1)
Qt version
5.15.3
Python version
3.9.5
Compiled against GDAL/OGR
3.5.2
Running against GDAL/OGR
3.5.3
PROJ version
9.1.0
EPSG Registry database version
v10.074 (2022-08-01)
GEOS version
3.10.3-CAPI-1.16.1
SQLite version
3.39.4
PDAL version
2.4.3
PostgreSQL client version
14.3
SpatiaLite version
5.0.1
QWT version
6.1.6
QScintilla2 version
2.13.1
OS version
Windows 10 Version 2009

Active Python plugins
attributeBasedClustering
2.2
awap
2.0.1
coordinate_capture
0.2
d3datavis
3.0.3
dataexplorer
3.1.1
DataPlotly
3.9.2
GeoCoding
2.18
gml_application_schema_toolbox
1.4.0-beta6
Hqgis
1.1.1
latlontools
3.6.6
layout_loader
1.1
lizmap
3.9.2
mmqgis
2021.9.10
NavTable
0.1.1
networks
2.7.3
nominatim_locator_filter
0.2.4
OnlineRoutingMapper
0.9
openlayers_plugin
2.0.0
OSMDownloader
1.0.3
OSMtools
3.2
pgRoutingLayer
3.0.2
QCopycanvas
0.5
qdraw
3.0.2
qgeric
1.91
qgiscloud
3.6.0
QuickOSM
2.1.1
SemiAutomaticClassificationPlugin
7.10.8
shapetools
3.4.9
timemanager
3.6
vector_tiles_reader
3.2.2
VoGisProfilTool
3.0.2
db_manager
0.1.20
grassprovider
2.12.99
MetaSearch
0.3.5
processing
2.12.99
sagaprovider
2.12.99

### Supported QGIS version

- [ ] I'm running a supported QGIS version according to the roadmap.

### New profile

- [ ] I tried with a new QGIS profile

### Additional context

Thank you for asking for the context
I am trying to setup a QGIS Server + QWC2 + QWC microservices web mapping website using Windows 10 Home. To that end trying to install whatever QGIS software is necessary for over 4 or 5 consecutive days but have not been successful.

I do appreciate the new documentation for QGIS Desktop/Server 3.22 LTR as well as the approach taken in "Advanced Install" mode of OSGeo4W to mimic package management in Windows. However, this is turning out to be too confusing for Windows users as well as those users not familiar with Linux based installations and it CLI. I have also seen the increasing clarity in the documentation and do gratefully appreciate that. 

Perhaps this may not be the right thread to mention the following issue, there seem to be some bugs in QGIS Server documentation https://docs.qgis.org/3.22/en/docs/server_manual/getting_started.html#installation-on-windows 

- Does the installation procedure create a folder named OSGeo4W64 ? This is suggested in **Section 2.2 Installation on Windows**
- Which folders are the *.qgs and the data stored? When installed through "Advanced Install", could not find the data for **Section 2.3 "Serving the Project"** 

Any help will be deeply appreciated and thank you in advance

**Comment by TerraTraveler** [comment]
Thank you so very much!
I Really appreciate your quick help!

**Comment by agiudiceandrea** [comment]
@TerraTraveler, have you managed to fix the issue? If so, please report back how and close this issue report.

**Comment by TerraTraveler** [comment]
I don't know if I am replying to the correct place, but yes, The issue was
fixed as suggested by Nicolas. Thanks again, Nicolas!
I used the screenshot of "Advanced Install" he provided to include the two
components. One of these two components (qgis-ltr-server was already
installed previously but the other one (qgis-ltr-full: QGIS Desktop Full
(meta package; long term release)) was not.

I did not do a 'clean install' as suggested, which I will be attempting
next (as this one may be related to a typo in the documentation of QGIS
Server 3.22 manual in section 2.2 Installing
